tests/sync_tests/requests_test.py

```diff
@@ -0,0 +1,172 @@
+import httpx
+import pytest
+from unittest.mock import MagicMock, patch
+
+from ctfdpy.ctfdpy.http import HTTPClient, HTTPMethod
+from ctfdpy.ctfdpy.exceptions import HTTPError
+
+@pytest.mark.anyio
```

Comment:
AsyncIO is used instead of AnyIO, which is an external library.

In addition, with some configs, you can avoid these instructions.
https://stackoverflow.com/questions/57461868/what-is-pytest-mark-asyncio-is-used-for
https://pytest-asyncio.readthedocs.io/en/latest/reference/configuration.html